aip/general/0203.md

```diff
@@ -42,6 +42,14 @@ backwards-compatility. Nontheless, this annotation **must not** be omitted.
 only, and does not itself add any validation. The purpose is to consistently
 document this behavior for clients.
 
+### Annotating Name
+
+The `name` field of a resource **must not** be annotated.
```

Comment:
I'm wondering if we should start by making this a **should not** instead, so as to avoid randomizing anyone that has already annotated it with field_behavior. Also, I'm worried that if we do end up adding a new value like `IDENTITY` we will be reverting this **must not** to "**must** use `IDENTITY`", further randomizing.